tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1849,6 +1838,94 @@ class ParameterizedOpConverterTestBase
 };
 
 template <typename T>
+class OpConverter_UnaryTest : public ParameterizedOpConverterTestBase {
+ public:
+  template <typename S>
+  void RunTests(
+      const string& testName, std::vector<string>& ops_to_test,
+      const operationMap<S>& map,
```

Comment:
Shouldn't the type be T?

Reply:
No, we need both **T** and **S. T** is used for types (float, int,, bool etc) and **S** is used for enums (`nvinfer1::UnaryOperatrion, nvinfer1::ActivationType`) used in `operationMap<S>`